src/component/BpmnVisu.ts

```diff
@@ -13,15 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import MxGraphConfigurator from './mxgraph/MxGraphConfigurator';
 import { mxgraph } from 'ts-mxgraph';
 import { defaultMxGraphRenderer } from './mxgraph/MxGraphRenderer';
 import { defaultBpmnParser } from './parser/BpmnParser';
-import { MxGraphFactoryService } from '../service/MxGraphFactoryService';
+
+declare const mxClient: typeof mxgraph.mxClient;
+declare const mxUtils: typeof mxgraph.mxUtils;
 
 export default class BpmnVisu {
-  private mxClient: typeof mxgraph.mxClient = MxGraphFactoryService.getMxGraphProperty('mxClient');
-  private mxUtils: typeof mxgraph.mxUtils = MxGraphFactoryService.getMxGraphProperty('mxUtils');
+  private mxClient: typeof mxgraph.mxClient = mxClient;
+  private mxUtils: typeof mxgraph.mxUtils = mxUtils;
```

Comment:
:question: I have tested that we can get rid of all private field related to mxGraph globals.
That way, we can simplify the code using them everywhere in the code
For instance, instead of `this.mxClient.isBrowserSupported()`, we can use `mxClient.isBrowserSupported()`

As a reminder, we introduced private field to avoid calling the `MxGraphFactoryService` on file load and did that at classes initialization instead (mainly to avoid side effects when running unit tests where mxGraph globals weren't initialized)

Reply:
I actually started to do it in next PR, but no problem can be done here already